src/org/opensha/sha/earthquake/faultSysSolution/ruptures/util/SectionDistanceAzimuthCalculator.java

```diff
@@ -123,7 +123,7 @@ public double getAzimuth(int id1, int id2) {
 	}
 	
 	public ConcurrentMap<IDPairing, Double> getCachedAzimuths() {
-		return distCache.asMap();
+		return azCache.asMap();
```

Comment:
@voj this looks like a bug got squashed. Did this get exposed through your new tests?